versions.tf

```diff
@@ -1,10 +1,10 @@
 terraform {
-  required_version = "~> 1.3"
+  required_version = ">= 1.10"
```

Comment:
Please don't modify the TF version requirement: this module doesn't require Terraform v1.10 as it doesn't use any features introduced in that version. While v1.3 is outdated, it should work with this module (even v1.0 should work as far as I can tell).

Additionally, our internal pipelines haven't moved to TF v1.10 yet, so requiring v1.10 for this module could be a problem when consumed in our internal pipelines.